chapters/classes.tex

```diff
@@ -1500,6 +1500,12 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The default \lstinline!min! and \lstinline!max!-attributes are the minimum and maximum representable finite floating point numbers of the underlying type.
+The value must be a finite floating point number, and thus the default default \lstinline!min! and \lstinline!max!-attributes do not further constrain the value.
+\begin{nonnormative}
```

Comment:
Just for better source code readability:
```suggestion

\begin{nonnormative}
```

Reply:
Did that as part of update